### PR TITLE
Feature 110 (unitest for mysql_errordetector)

### PR DESCRIPTION
**Pull request by gmichalo**

### CODE REVIEW

File: test/Unit_tests/unittest_mysql_dcerrordetector.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+    def test_number_of_dk_cells(self):
+
+        self.session.load_denial_constraints(
+            "../../datasets/unit_test/unit_test_constraints.txt")
+
+        detector = MysqlDCErrorDetection(self.session.Denial_constraints,
+                                         holo_obj,
+                                         self.session.dataset)
+        self.session.detect_errors(detector)
```
Comment (j48zheng) [inline test/Unit_tests/unittest_mysql_dcerrordetector.py]: We make an error detector for every testcase, so put it in setup

**Review by j48zheng — CHANGES_REQUESTED** [review]

File: test/Unit_tests/unittest_mysql_dcerrordetector.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import unittest
```
Comment (JVonderwell) [inline test/Unit_tests/unittest_mysql_dcerrordetector.py]: unrelated to the code itself but there's already a 'unit_tests' folder. we should pick one name or the other and stick with it.

**Review by JVonderwell — DISMISSED** [review]
looks good, just see folder naming comment.

**Review by JVonderwell — DISMISSED** [review]

**Review by j48zheng — APPROVED** [review]

**Review by j48zheng — CHANGES_REQUESTED** [review]
Any code that's at the beginning of all test cases should go into setup

**Review by j48zheng — DISMISSED** [review]

**Review by ah89 — DISMISSED** [review]

**Review by jw-mcgrath — DISMISSED** [review]

**Pull request closed without merging**